### PR TITLE
Add Renovate config

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Do less Ops and more Dev.

### CODE REVIEW

File: internal/renovate/renovate.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 SAP SE
```
Comment (talal) [inline internal/renovate/renovate.go:1]: The copyright year should be 2022.

**Review by talal — CHANGES_REQUESTED** [review]
I am not familiar with Renovate Bot so I can't really comment on the config structure.

I think @majewsky did some reading on Renovate Bot so if you want to get his feedback then add him as reviewer otherwise feel free to go ahead with the merge.

File: internal/core/config.go [hunk 2]
```diff
@@ -45,6 +45,7 @@ type Configuration struct {
 	GolangciLint   GolangciLintConfiguration    `yaml:"golangciLint"`
 	SpellCheck     SpellCheckConfiguration      `yaml:"spellCheck"`
 	GitHubWorkflow *GithubWorkflowConfiguration `yaml:"githubWorkflow"`
+	Renovate       RenovateWorkflowConfig       `yaml:"renovate"`
```
Comment (talal) [inline internal/core/config.go:48]: I missed this earlier. The struct should not have workflow in its name as it is not a workflow.

**Review by majewsky — APPROVED** [review]
As discussed in Slack, please track bugfix releases for golang and alpine to simplify evidence collection for our patch management process. Otherwise, this is good to go in.

**Pull request merged**